package.json

```
@@ -24,7 +24,7 @@
     "eslint-config-airbnb-typescript": "^8.0.2",
     "eslint-config-prettier": "^6.11.0",
     "eslint-plugin-fp": "^2.3.0",
-    "eslint-plugin-import": "^2.21.2",
+    "eslint-plugin-import": "^2.22.1",
```

```suggestion
    "eslint-plugin-import": "2.22.1",
```

I think you'll need to pin it? (I could be wrong, I always get a bit confused by npm versioning syntax)